docs/_onboard/seeding-the-cosmos.md

```diff
@@ -9,9 +9,9 @@ last_modified_date: 2022-12-01
 Seeding the Cosmos
 =======================
 
-Y-Foundry DAO is a community born out of frustration with the often extractive and insecure nature of crypto. Manipulation infects systems advertised as trustless, leading to unintended consequences and losses. Cults of personalities develop and take advantage of the uninformed.
+Y-Foundry DAO is a community born out of frustration and contempt with opague and exploitative crypto projects. These projects are often run by cult-like personalities who extracts value from uninformed participants to enrich their own pockets, while avoiding transparency and accountability for their actions and decisions.
```

Comment:
"opague" should be "opaque"
Not sure if "cult-like personalities" is accurate. Cult-like would be describing the followers not the leaders. Maybe "personalities who abuse a cult-like following with their uninformed participants to enrich their own pockets..."

Reply:
Discussed with Ray - this is an old page and is not longer in /docs. should be removed